2-Draft-Accepted/RFCXXXX-MSI-Installation-Path.md

```diff
@@ -42,9 +42,10 @@ We also provide the version of PowerShell in the FileVersion object associated w
   an in-place update of that stable build shall be performed.
 * When a preview MSI is run on a machine that already has a preview PowerShell Core 6.x installed,
   an in-place update of that preview build shall be performed.
-* No installation of a preview build shall interfere with a stable build, and vice versa
+* By default, no installation of a preview build shall interfere with a stable build, and vice versa
     * This includes an implicit requirement that `Install-PowerShellRemoting.ps1` create a different endpoint name for the preview build.
-    * The preview binary of PowerShell Core 6.x shall be named `pwsh-preview` or `pwsh-preview.exe` (depending on the platform).
+    * A UI option will be added to the MSI to add the product to the path.
+      The option will be defaulted on for stable builds and off for preview builds.
```

Comment:
Should we uninstall a version installed to the path without product path? (Can we use "system-wide installation" term?)
If we should what about that stable and preview builds have different product update GUIDs?